.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: go
+go:
+    - 1.6.x
+    - 1.7.x
+    - 1.8.x
+    - tip
+
+sudo: required
+
+install:
+    - make install.tools
+
+before_script:
+    - export PATH=$HOME/gopath/bin:$PATH
+
+script:
+    - make .gitvalidation
```

Comment:
It seems that the commit 022b25259e31784d68192cdcd87a82d6c357a1bb doesn't have DCO, and will cause error if we run locally.

@mikebrow For this, should we update the previous commit or skip it? Maybe we should change https://github.com/kubernetes-incubator/cri-containerd/blob/master/Makefile#L2 to 022b25259e31784d68192cdcd87a82d6c357a1bb.